Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -347,8 +339,11 @@ ModelExperimentalSceneGraph.prototype.buildDrawCommands = function (
       runtimePrimitive.drawCommands = drawCommands;
     }
   }
-  this._boundingSphere = BoundingSphere.fromBoundingSpheres(
-    this._boundingSpheres
+
+  this._boundingSphere = BoundingSphere.fromBoundingSpheres(boundingSpheres);
+  this._model._boundingSphere = BoundingSphere.transform(
+    this._boundingSphere,
+    this._model.modelMatrix
```

Comment:
here you can pass in `this._model._boundingSphere` as the third parameter so it gets updated in place if it was already created instead of allocating a new one.